src/cephadm/cephadm

```diff
@@ -2418,7 +2418,8 @@ def command_bootstrap():
         # make sure IP is configured locally, and then figure out the
         # CIDR network
         for net, ips in list_networks().items():
-            if base_ip in ips:
+            if ipaddress.ip_address(unicode(base_ip)) in \
+                    [ipaddress.ip_address(unicode(ip)) for ip in ips]:
```

Comment:
Ipv6 address string could come in shortened in different way. So we use ipaddress to normalise it. The unicode is for py2 support.